media/system/js/share-uncompressed.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @package         Joomla.JavaScript
+ * @copyright       Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+/**
+ * Calls the sending process of the config class
+ */
+(function($){
+$(document).ready(function ($){
+    $('#sharebutton').click(function ()
+    {
+        var share_draft = {
+            articleId  : $('#jform_id').val(),
+            title: $('#jform_title').val(),
+
+        };
+
+        // Remove js messages, if they exist.
+        Joomla.removeMessages();
+
+        $.ajax({
+                url: 'index.php?option=com_content&task=article.shareDraft',
+                data: share_draft,
+                dataType: "json"
+            })
+            .done(function (response) {
+                // Render messages, if any.
+                if (typeof response.messages == 'object' && response.messages !== null)
+                {
+                    Joomla.renderMessages(response.messages);
+
+                    window.scrollTo(0, 0);
+                }
+
+    });
+});
+}(jQuery);
+
```

Comment:
This is a JavaScript file, so you don't need that here. You need to load it (from your view) with

JHtml::_('script', 'path/to/file' ...)

See Rolands sample here: 

https://github.com/joomla-extensions/boilerplate/commit/64d594472412086a1a6a7a492ff45e53a5538a00